Quantum-Control-Applications/Superconducting/Single Flux Tunable Transmon/SSB mixer for readout/README.md

```diff
@@ -0,0 +1,25 @@
+# Single flux tunable transmon with SSB mixer for readout
+
+<img align="right" src="SIngle Flux Tunable Transmon Setup.PNG" alt="drawing" width="400"/>
+
+## Experimental setup and context
+
+These files showcase various experiments that can be done on an single flux tunable transmon where the readout pulses 
+are sent through a SSB mixer. Standard IQ mixers are used for qubit adressing and signal downconversion before acquisition.
+
+These files were tested in a real setup shown on on the right, but are given as-is with no guarantee.
+
+While these can serve as a template for new labs or for new experiments, certain adaptations will probably have to be made.
+Use with care.
+
+## Basic Files
+1. [Hello_qua](hello_qua.py) - A script used for basic qua program demonstration.
+2. [Look_at_raw_traces](look_at_raw_traces.py) - A script for acquiring raw ADC traces from inputs 1 and 2 and check ADC saturation and time of flight.
+3. [Resonator_spec](resonator_spec.py) - Performs the 1D and 2D (with flux amplitude sweep) resonator spectroscopy.
+4. [Rabi_amp_freq](rabi_amp_freq.py) - Acquires the 2D (pulse amplitude & frequency sweeps) Rabi oscillations.
+5. [ramsey_freq_duration](ramsey_freq_duration.py) - Acquires the 2D (idle time & pulse frequency sweeps) Ramsey oscillations.

```

Comment:
Right now, just the resonator spectroscopy has the 1D and 2D options. All the others are only in 2D

Reply:
Ok, cool.
Do we want to add the 1D options?

Reply:
I could but since there are 3 different possible 1D scans it feels a bit cumbersome...
On my way to DAMOP, I created a class that contains all the basics calibrations (Spectroscopy, Rabi, Ramsey, TOF...) that can be easily called and set with 1D or 2D scans of amplitude, duration and frequency. The idea is to have an easy calibration tool leaning the path towards Entropy. I still need to thoroughly test it but I'll probably present it at the roles meeting this afternoon.

Reply:
Nice!

I see your point.
My only objection to "only 2d scan" is that these are a waste of time before the 1D works. So I think we only need the 1D for frequency.
We can also make it such that the plotting automatically sees how many points you have in each dimension and correctly plots it. (Easier to explain in person)

Reply:
I am not sure I understand what you mean, but I'll be happy to discuss it over a call when you have some time :)

Reply:
nevermind for now, maybe in the future, let's push the other things so we'll be able to release it

Reply:
Alright so if you don't have any more comments I think we can approve and merge